README.md

```diff
@@ -266,10 +266,15 @@ Another example is the Node.js [unofficial-builds project](https://github.com/no
 
     export N_NODE_MIRROR=https://unofficial-builds.nodejs.org/download/release
 
-You may need to specify the architecture explicitly if not autodetected by `n`, such as using `musl` `libc` on Alpine:
+You may need to specify the architecture explicitly if not autodetected by `n`, such as using `musl` `libc` on Alpine. You can do that with using `N_ARCH`:
```

Comment:
I think fine to just have `N_ARCH` covered in example here:

```suggestion
You may need to specify the architecture explicitly if not autodetected by `n`, such as using `musl` `libc` on Alpine. You can do that with `N_ARCH` or `--arch`:
```

Reply:
Combined both, see the new patch which I'll push shortly.